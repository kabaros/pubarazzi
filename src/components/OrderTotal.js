import React from 'react';
import { calculateTotal } from '../helpers/roundsUtils';

export default ({ orders }) => {
    const total = calculateTotal(orders);
    return (<header className="order-total">
        Total: Â£{total}
    </header>);
}