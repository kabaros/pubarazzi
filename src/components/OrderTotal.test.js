import React from 'react';
import { render } from 'react-dom';
import OrderTotal from './OrderTotal';

it('renders default title', () => {
    const div = document.createElement('div');
    const orders = {
        drink1: {
            count: 2,
            price: 10
        },
        drink3: {
            count: 1,
            price: 20
        }
    }
    render(<OrderTotal orders={orders} />, div);

    expect(div).toMatchSnapshot();
});

