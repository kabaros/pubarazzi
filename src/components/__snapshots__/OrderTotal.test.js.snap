// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default title 1`] = `
<div>
  <header
    class="order-total"
  >
    Total: Â£
    40
  </header>
</div>
`;
